### PR TITLE
[stable/postgresql] Allow a template for persistence.existingClaim

### PR DESCRIPTION
**Pull request by michaelbeaumont**
#### What this PR does / why we need it:

Allows `.Values.persistence.existingClaim` to depend on things like `.Release.Name`.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by jonninen50 — APPROVED** [review]

File: stable/postgresql/values.yaml [hunk 1]
```diff
@@ -197,6 +197,7 @@ persistence:
   enabled: true
   ## A manually managed Persistent Volume and Claim
   ## If defined, PVC must be created manually before volume will be bound
+  ## The value is evaluated as a template, so, for example, the name can depend on .Release or .Chart
```
Comment (juan131) [inline stable/postgresql/values.yaml:200]: Please update `values-production.yaml` too.

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for the contribution @michaelbeaumont 

I'm okay with the changes but please extend the changes to `values-production.yaml` too.

File: stable/postgresql/values.yaml (hunk 1 above)
Comment (michaelbeaumont) [reply stable/postgresql/values.yaml:200]: done, nice catch

**Pull request merged**